container/dependencies.nix

```diff
@@ -1,11 +1,20 @@
-{ pkgs }:
+{ pkgs, nvDrivers, defaultNvDriver }:
 
 with pkgs;
 
 let patched_rPackages =
    rPackages.override { overrides = {
      xml2 = rPackages.xml2.overrideDerivation (a: { installFlags = a.installFlags ++ ["--no-lock"]; })
-   ;} ;};
+     ;} ;};
+    relevant_drivers =
+      with builtins; filter (e: elem e.version (nvDrivers ++ [defaultNvDriver])) nixgl.knownNvidiaDrivers;
```

Comment:
doesn't this check for the default driver twice ?

Reply:
yes, just a contingency if someone forgot to put the default driver in the list of supported drivers